.github/workflows/ci.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install .[test]
-        pip install https://github.com/bboe/coveralls-python/archive/github_actions.zip
+        pip install git+https://github.com/coveralls-clients/coveralls-python.git@6064a81
```

Comment:
Why tag a specific commit? Don't we want to allow newer versions if needed?

Reply:
bboe's fork is stuck on this commit, so to ensure continuity, I pinned the clone as well.

Reply:
I'd rather not pin this specific commit because it wasn't previously pinned. Since bboe has control over his fork, there won't be any unexpected breaking changes that PRAW contributors don't know about.